### DISCUSSION TITLE
请问哪个版本支持Sql Server 2000

### DISCUSSION THREAD

**Discussion by neozhu** [dotnetcore/FreeSql#1432] [General]
.net core已经放弃sql 2000了，我现在有一个需求需要读取Sql server 2000 我看了文档最低支持 Sql Server 2005, 请问哪个版本可以支持SQL 2000

**Comment by 2881099** [comment]
freesql.provider.custom

**Reply by neozhu** [reply]
thank you

**Reply by 2881099** [reply]
可能尝试用 odbc 或者 oledb 访问
